source/s3_request_messages.c

```diff
@@ -539,7 +588,7 @@ struct aws_http_message *aws_s3_complete_multipart_message_new(
             goto error_clean_up;
         }
 
-        aws_s3_message_util_assign_body(allocator, body_buffer, message);
+        aws_s3_message_util_assign_body(allocator, body_buffer, message, AWS_SCA_NONE, NULL);
```

Comment:
trivial: we usually do things like `aws_s3_message_util_assign_body(allocator, body_buffer, message, AWS_SCA_NONE, NULL/* out_checksum */);`